vertexai/token-count/go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/GoogleCloudPlatform/golang-samples/vertexai/token-count
```

Comment:
Maybe something has changed since I last looked, but as I recall the trend was heading towards a single module for snippets per product, with module-per-sample being limited to sample apps.

@muncus do you have feedback?

Reply:
@grayside this is consistent with all the samples inside `vertexai`.

I'd be happy to change the module approach for all of `vertexai` in a subsequent PR, if we decide to move that way